vo_new/components/charts/RechartsComponent.tsx

```
@@ -0,0 +1,272 @@
+      name="Data Points" 
+      data={chartData} 
+      fill="#ef4444" 
+    />
+  </ScatterChart>
+);
+
+const renderUnsupportedChart = (chartType: string) => (
+  <div className="flex items-center justify-center h-full">
+    <div className="text-center">
+      <div className="text-muted-foreground mb-2">
+        Chart type "{chartType}" is not yet supported in Recharts
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)